be/src/http/action/stream_load.cpp

```diff
@@ -408,9 +408,6 @@ Status StreamLoadAction::_process_put(HttpRequest* http_req,
     if (!http_req->header(HTTP_TIMEZONE).empty()) {
         request.__set_timezone(http_req->header(HTTP_TIMEZONE));
     }
-    if (!http_req->header(HTTP_TIME_ZONE).empty()) {
```

Comment:
why delete time zone header? We do not support time zone in stream load any more？

Reply:
that was added by mistake. as we set in document, we should use 'timezone' not 'time_zone'